gulpfile.js

```diff
@@ -562,11 +559,11 @@ function buildPlatform(cb) {
             ])
             .pipe(gulp.dest("dist/" + platformName))
             .on("end", function () {
-                if (platformName == "Angular") {
-                    log("building " + PLAT + " ... done ");
-                    log("=========================================================");
-                    cb();
-                } else {
+                // if (platformName == "Angular") {
+                //     log("building " + PLAT + " ... done ");
+                //     log("=========================================================");
+                //     cb();
+                // } else {
```

Comment:
This check was excluding Angular from the build. In order to enable the CI for Angular, I needed the Angular build to go through the pipeline same way as all the other platforms.

Reply:
we cannot comment out that IF statement because it would generate docfx files (see highlighted files in attached image) in dist/output folder. For Angular, we should have only images and markdown files because these files are submitted as a PR to [Agnular docs repo](https://github.com/IgniteUI/igniteui-docfx) and generated docfx files would override files in [Angular docs repo](https://github.com/IgniteUI/igniteui-docfx). Can you undo commenting out that IF statement? Later we can figure out how to create Angular CI build in xplat-docs repo.

![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/11665978/255d0900-639d-44f6-b454-fa87fe670e82))

Reply:
I've restored the commented out part and I've setup the Angular CI to just work with the .md files. The CI doesn't really validate anything for now, but it can be updated once the docfx build portion is setup.